app/controllers/graphql_controller.rb

```diff
@@ -11,12 +12,8 @@ def execute
       tracing_enabled: ApolloFederation::Tracing.should_add_traces(headers),
       current_user: current_user,
     }
-    result =
-      LupoSchema.execute(
```

Comment:
I was under the impression the apollo server we're using is already caching here.

Reply:
it is definitely NOT caching. Otherwise, visit the same page twice would be super faster, and it's super slow.